tests/pytorch/iris.py

```diff
@@ -1,23 +1,21 @@
-# pylint: disable=arguments-differ
```

Comment:
Could you rervert changes on files under `tests`?

Reply:
sure . i will revert the test files.

Reply:
done. i have reverted those files